inc/api.php

```diff
@@ -229,6 +267,23 @@ public function add_watermark( $file ) {
 
 		return $response;
 	}
+	/**
+	 * Call the images endpoint.
+	 *
+	 * @param string|false $scroll_id Scroll id used to advance the search.
+	 * @return mixed The decoded json response from the api.
+	 */
+	public function get_cloud_images( $scroll_id = false ) {
+		$params = ['key' => Optml_Config::$key ];
+		if ( $scroll_id === false ) {
+			$scroll_id = get_transient( 'scroll_id' );
+			delete_transient( 'scroll_id' );
+		}
+		if ( false !== $scroll_id ) {
+			$params['scroll_id'] = $scroll_id;
```

Comment:
@GrigoreMihai what exactly is `scroll_id`? Is that the page number? If so, let's use `page` instead of `scroll_id` since it's more clear.

Reply:
It is coming from the api response, so in the next call to the api we can send it to get the next images, it something like page number I will change it to `api_page` since I already have a `page` to keep track of the wp modal paging

Reply:
Let's use plain simple `page`